### ISSUE TITLE
Code never used

### ISSUE THREAD

**Issue by afarrellsherman** [afarrellsherman/Woolf#13]
This code isn't used because `feature table` is a required argument.  Should this be fixed?

https://github.com/afarrellsherman/Woolf/blob/f87c3fda74bb8d75486f77d4d3ac20c3194c9822/bin/trainWoolf#L67

**Comment by kescobo** [comment]
Yeah, `argparse` handles stuff like this for you. You shouldn't have any code to manually deal with command line arguments.
